### PR TITLE
Update and rename Default.java to Greek.java

### PR DESCRIPTION
**Pull request by nikosbks**

### CODE REVIEW

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java [hunk 1]
```diff
@@ -1,214 +1,215 @@
...
++import org.tillerino.osuApiModel.OsuApiUser;
++
++import tillerino.tillerinobot.BeatmapMeta;
++import tillerino.tillerinobot.IRCBot.IRCBotUser;
++import tillerino.tillerinobot.RecommendationsManager.Recommendation;
++
++public class Default implements Language {
++
++	@Override
++	public String unknownBeatmap() {
```
Comment (TheHowl) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java]: Still seems to be greek even in Default.java.

Comment (TheHowl) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java]: Also, what are all those pluses doing there?

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Greek.java [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+	@Override
+	public String unknownBeatmap() {
+		return "Συγνώμη, δεν γνωρίζω αυτό το τραγούδι. Ίσως είναι αρκετά νεο, πολυ δύσκολο, μη εγκεκριμένο ή να μην είναι για το osu standard mode." ;
+	}
+
+	@Override
+	public String internalException(String marker) {
+		return "Εχ... μάλλον φαίνεται ότι ο ανθρώπινος Tillerino έκανε μαντάρα την σύνδεσή μου."
+				+" Εάν δεν το παρατηρήσει σύντομα, μπορείς [https://github.com/Tillerino/Tillerinobot/wiki/Contact να τον ενημερώσεις];" (reference "
```
Comment (TheHowl) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Greek.java]: uh? What happened here (and in a lot of lines down there)? If you want to insert `"`, you need to do `\"` to make java  understand you want to print a `"`, not to make the string end.

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Greek.java [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+	}
+
+	@Override
+	public String invalidAccuracy(String acc) {
+		return "Άκυρη ακρίβεια: \"" + acc + "\"";
+	}
+
+	@Override
+	public void optionalCommentOnLanguage(IRCBotUser user, OsuApiUser apiUser) {
+		user.message("Οπότε σου αρέσω ακριβώς όπως είμαι :)");
```
Comment (Tillerino) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Greek.java]: Sorry, this was not in the guide yet:

The optionalCommentOnLanguage message is a special one. The bot sends it after you picked this language. Translating the English message makes no sense. Most translations use "YOUROSUNAME helped me learn YOURLANGUAGE" here.

**Comment by Tillerino** [comment]
I guess I'll ignore all the changes in Default.java, right? So what will be the name of the language? I need something for the command, so like `!set lang ελληνικά`

**Comment by nikosbks** [comment]
Yea ignore all the changes in Default.java . The name of the language is Ελληνικά , so you are right the command should be !set lang Ελληνικά.

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Greek.java (hunk 3 above)
Comment (nikosbks) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Greek.java]: I'm sorry but I don't really get what you mean there.

Comment (Tillerino) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Greek.java]: "So you like me just the way I am" refers to the user picking the *Default* Language implementation. I suggest putting "nikosbks helped me learn Greek" here (but in Greek and with your osu name, if that's not your osu name as well). It gives credit and acknowledges the language change.

Comment (nikosbks) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Greek.java]: Oh I see. Okey.

**Comment by Tillerino** [comment]
Translation should be available now.

**Pull request closed without merging**